pom.xml

```diff
@@ -146,8 +146,8 @@ The following provides more details on the included cryptographic software:
     <target.name>all</target.name>
     <maven-antrun-plugin.version>1.8</maven-antrun-plugin.version>
     <junit.version>4.12</junit.version>
-    <commons-logging.version>1.1.3</commons-logging.version>
-    <commons.jacoco.version>0.7.7.201606060606</commons.jacoco.version>
+    <commons-logging.version>1.2</commons-logging.version>
+    <commons.jacoco.version>0.8.0</commons.jacoco.version>
```

Comment:
Again, I am unsure why these version bumps are necessary. They could be explained in the commit message.